src/module-server/middleware/js.ts

```diff
@@ -153,29 +206,18 @@ export const jsMiddleware = ({
         'Content-Length': Buffer.byteLength(code, 'utf-8'),
       });
       res.end(code);
-
-      // Start a esbuild build (just for the sake of parsing)
-      // That way, if there is a parsing error in the code resulting from the rollup transforms,
-      // we can display an error/code frame in the console
-      // instead of just a generic message from the browser saying it couldn't parse
-      // We are *not awaiting* this because we don't want to slow down sending the HTTP response
-      esbuild.transform(code, { loader: 'js' }).catch((error) => {
-        const err = error.errors[0];
-        const { line, column } = err.location;
-        const frame = createCodeFrame(code as string, line - 1, column);
-        const message = `${colors.red(colors.bold(err.text))}
-
-${colors.red(`${id}:${line}:${(column as number) + 1}`)}
-
-${frame}
-`;
```

Comment:
I deleted this feature. It is possible that I will add it back in the future, but it is definitely an edge-case and I didn't want to slow down _every_ request by 10-300ms because of it.

This would _only_ catch syntax errors before they got to the browsers if all of these are true:
- The user writes invalid code and they don't have the regular esbuild transform enabled, or a user-added plugin turns valid code invalid after esbuild.
- es-module-lexer thinks the end result code is valid (es-module-lexer is kinda a loose parser so this does happen sometimes)